test_data/makedata_suites/610_analyzers.js

```diff
@@ -0,0 +1,255 @@
+/* global print, semver, progress, createSafe, db, PWD*/
+/*jslint maxlen: 130 */
+
+(function () {
+  const a = require("@arangodb/analyzers");
+  return {
+    isSupported: function (currentVersion, oldVersion, options, enterprise, cluster) {
+      let currentVersionSemver = semver.parse(semver.coerce(currentVersion));
+      let oldVersionSemver = semver.parse(semver.coerce(oldVersion));
+      return semver.gte(oldVersionSemver, "3.10.0");
+    },
```

Comment:
Technically good, but we could export that method to a global helper location area. 
I bet it is used on a lot of places.

Reply:
isSupported is individual per testcase - its part of the API which the tests have to implement.
createAnalyzer / checkAnalyzer /  deleteAnalyzer - yes. but in a subsequent PR.

Reply:
``` let currentVersionSemver = semver.parse(semver.coerce(currentVersion));
      let oldVersionSemver = semver.parse(semver.coerce(oldVersion));
      return semver.gte(oldVersionSemver, "3.10.0");```
      
      But this will always be identical, isn't it ? Except the version which is "3.10.0" in that specific case.

Reply:
no. other files have other logic to enable / disable the test. different versions, enterprise community, oldversion / newversion.
Only 607/608/609/610 follow the rule to be matching the arangodb minor version.